### PR TITLE
Added kind build node --type limitations

### PR DESCRIPTION
**Pull request by alejandrox1**
/kind documentation

Signed-off-by: Jorge Alarcon Ochoa <alarcj137@gmail.com>

### CODE REVIEW

File: site/content/docs/user/quick-start.md [hunk 1]
```diff
@@ -137,6 +137,10 @@ Currently, kind supports three different ways to build a `node-image`: via
 `apt`, or if you have the [Kubernetes][kubernetes] source in your host machine
 (`$GOPATH/src/k8s.io/kubernetes`), by using `docker` or `bazel`.
 To specify the build type use the flag `--type`.
+Note however that using `--type=bazel` in Windows or Mac will not work as some
```
Comment (BenTheElder) [inline site/content/docs/user/quick-start.md]: let's say "will not work currently" as this is a solve-able problem, it just isn't yet. it is also only kubelet.

```
Note however that using `--type=bazel` on Windows or MacOS will not work currently due to Kubelet using [CGO] which requires GCC/glibc for linux. A workaround may be enabled in the future.


[CGO]: https://golang.org/cmd/cgo/
```

We won't mention it here, but test-infra/planter integration is one possible option to explore later.

Comment (alejandrox1) [reply site/content/docs/user/quick-start.md]: Fixed

**Pull request merged**